src/backend/utils/misc/guc_gp.c

```diff
@@ -772,6 +772,17 @@ struct config_bool ConfigureNamesBool_gp[] =
 		NULL, NULL, NULL
 	},
 
+	{
+		{"gp_enable_explain_rows_out", PGC_USERSET, CLIENT_CONN_OTHER,
+			gettext_noop("Experimental feature: print avg, min and max rows out in segments in EXPLAIN ANALYZE."),
```

Comment:
it is not Experimental, is it feature from GP5

Reply:
Fixed